src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -24,9 +25,15 @@ internal static class NamedPipeUtils
     {
         #region Strings
 
-        internal const string DefaultAppDomainName = "DefaultAppDomain";
+        internal const string DefaultAppDomainName = "Default";
         internal const string NamedPipeNamePrefix = "PSHost.";
+#if UNIX
+        internal const string NamedPipeNamePrefixSearch = "CoreFxPipe_PSHost*";
```

Comment:
I guess dotnet adds the `CoreFxPipe_` prefix, right?

Reply:
I found it here https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs#L33

Reply:
Yes. We have the same logic in PowerShell Editor Services to handle the prefix. 

Funny how there's a length limit to the name of the pipe and CoreFX adds this silly prefix.